crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -70,9 +70,18 @@ def commit_and_check_proposal
   end
 
   def shutdown_services_at_non_db_nodes
-    db_nodes = []
-    NodeObject.find("state:crowbar_upgrade").each do |node|
+    upgrade_nodes = NodeObject.find("state:crowbar_upgrade")
 
+    # This is the first action on nodes from current version of product:
+    # ensure the nodes are reachable
+    upgrade_nodes.each do |node|
+      ssh_status = node.ssh_cmd("")
+      if ssh_status[0] != 200
+        raise I18n.t("model.service.node_not_available", name: node.name)
```

Comment:
Can we possible collect all nodes for which it's failing and display one message for all of them, instead of one message just for the first one?